sherpa-onnx/csrc/offline-nemo-enc-dec-ctc-model-config.h

```diff
@@ -12,11 +12,15 @@ namespace sherpa_onnx {
 
 struct OfflineNemoEncDecCtcModelConfig {
   std::string model;
+  const char *model_buf_begin, *model_buf_end;
```

Comment:
Please update the `Validate()` method to check that
when `model_buf_begin` is not nullptr, then model must be empty and vice versa.